packages/bpk-component-content-cards/README.md

```diff
@@ -0,0 +1,43 @@
+# BpkContentCards
+
+> Backpack example component.
+
+- [Storybook link](https://backpack.github.io/storybook)
+- [Content Cards Docs](backpack.github.io/components/content-cards)
+
+## Installation
+
+```sh
+npm install bpk-component-content-cards --save-dev
+```
+
+## Usage
+
+```tsx
+import React from 'react';
+import BpkContentCards from '@skyscanner/backpack-web/bpk-component-content-cards';
+
+<BpkContentCards
+    headerText='Header'
```

Comment:
sorry 😅 

Will it make sense to change to `headingText` or just `heading`? `headerText` seems to be a bit off, but maybe it's my head that has weird image of what 'headers' are

Reply:
Changed to `heading` 👍